.goreleaser.yml

```diff
@@ -0,0 +1,39 @@
+before:
+  hooks:
+    - cd _build && go mod download
+builds:
+  - id: opentelemetry-collector
+    dir: _build
+    flags:
+      - -trimpath
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm
+      - arm64
+      - s390x
+    goarm:
+      - 6
+      - 7
+dockers:
+  - image_templates:
+    - "quay.io/observatorium/opentelemetry-collector:{{ .Tag }}"
+    - "quay.io/observatorium/opentelemetry-collector:v{{ .Major }}"
```

Comment:
Why is the `v` prefix being added to the version tags? Think it would be better without. Which also may mean removing from the Tag one above.

Reply:
That's from the gorelease docs:
https://goreleaser.com/customization/docker/#keeping-docker-images-updated-for-current-major

I don't have a strong preference, we should follow what's common practice. But as the previous comment, I wanted to get this working first :)